### PR TITLE
changes on payments page

### PR DESCRIPTION
**Pull request by sarveshpro**
- [x] Match payment history design changes by Sagar - https://www.figma.com/file/7UxtNI4aSOZfg4Brz1Nh0y/WebApp-User-Dashboard?node-id=533%3A9864
- [x] Project name in funding donation, bouquet, conservation
- [ ] Show transfer details for an offline pending donation (waiting for backend change)
- [x] Trim project name and show on hover
- [x] Replace beta tag with new
- [x] Gap between donations
- [x] Donation Note
- [x] color coded donation status
Screenshot 2021-12-16 at 3 36 23 PM

### CODE REVIEW

File: public/static/locales/en/me.json [hunk 1]
```diff
@@ -186,5 +186,13 @@
...
-  "bouquet-donation": "Bouquet Donation"
-}
\ No newline at end of file
+  "bouquet-donation": "Bouquet Donation",
+  "transferDetails": "Transfer Details",
+  "beneficiary": "Beneficiary",
+  "donationNote": {
+    "offline-offline": "Bank transfers usually take 1–5 business days to reach us. As soon as your transfer arrives we will send you a confirmation email with your receipts (and your certificate).",
+    "stripe-sepa_debit": "SEPA payments take 2-14 business days (usually 3-5 days). As soon as it arrives we will send you a confirmation email with your tree certificate and raise the tree counter.",
+    "stripe-sofort": "SOFORT payments take 2-14 business days (usually 2-3 days). As soon as it arrives we will send you a confirmation email with your tree certificate and raise the tree counter.",
```
Comment (norbertschuler) [inline public/static/locales/en/me.json:195]: Are these messages only used for tree donation speaking from certificates and tree counters? @sagararyal as I understood we now have to carefully check the type of donation when speaking about certificates or tree counters, right?

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/profile/history**](https://planet-webapp-6j76q5t9p-planetapp.vercel.app/profile/history) | [**/profile/recurrency**](https://planet-webapp-6j76q5t9p-planetapp.vercel.app/profile/recurrency) |
|:-:|:-:|
|<a href="https://planet-webapp-6j76q5t9p-planetapp.vercel.app/profile/history">Screenshot of /profile/history</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/SOR96D8ZyIxOgkIZtLDKtDf-qZj1Bzw6ytCPS426H0e7AaV5_JbnAqg2WDkIOXeDgq8SoIjO-yABWajwy6DpFg.png">(view full size)</a> |<a href="https://planet-webapp-6j76q5t9p-planetapp.vercel.app/profile/recurrency">Screenshot of /profile/recurrency</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/pomoX9cqW6HtYdNNylHnMrPe8F28RACq23MjoPeaoTS16wl8mOOQwGf2wvEzuitrKDCa3tQI6TTdW0k3Ly-Feg.png">(view full size)</a> |







Commit 64dc3bc7952e8b4e22415d2ff8711a14ac4211d7 (https://planet-webapp-6j76q5t9p-planetapp.vercel.app).

File: public/static/locales/en/me.json (hunk 1 above)
Comment (sagararyal) [reply public/static/locales/en/me.json:195]: @norbertschuler correct, this should be adapted for donation types.

**Review by sagararyal — CHANGES_REQUESTED** [review]
Fix Eslint errors, cypress tests and adapt the information text on bank transfers from donate.plant-for-the-planet.org for multiple purpose.

If swift = swift/ null, hide.

**Pull request merged**